### PR TITLE
ci: add github actions

### LINKED ISSUE #9: Add issue/PR template
See https://help.github.com/en/github/building-a-strong-community/about-issue-and-pull-request-templates

### LINKED ISSUE #10: Set up CI with GitHub Actions
`tendermint-rs` is moving from Circle CI to GitHub Actions: https://github.com/interchainio/tendermint-rs/pull/144

### PR DESCRIPTION
**Pull request by marbar3778**
- seperated audit check into its own file but testing, build and linting lives in one file.
#### - someone will need to add the token for codecov for ci to be successful
- added issue (bug & feature) templates along with a PR template

closes #10 
closes #9 

while speaking with zaki a few weeks ago he recommended stable in ci and local nightly, if you'd like nightly in ci let me know its a minor change
Signed-off-by: Marko Baricevic <marbar3778@yahoo.com>

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+are the most critical to review.
+-->
+
+
+______
+
+For contributor use:
+
+- [ ] Wrote tests
+- [ ] Updated CHANGELOG_PENDING.md
```
Comment (marbar3778) [inline .github/PULL_REQUEST_TEMPLATE.md]: not sure if you will have a pending file or just a changelog with a unreleased section

File: .github/workflows/rust.yml [hunk 2]
```diff
@@ -0,0 +1,80 @@
+name: Rust
+on: [pull_request]
+jobs:
+  cleanup-runs:
```
Comment (marbar3778) [inline .github/workflows/rust.yml]: there is no way to stop ci when a new commit is pushed currently built into GitHub, this is meant as a workaround for now.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (ancazamfir) [reply .github/PULL_REQUEST_TEMPLATE.md]: not sure, let's just use the latter.

**Comment by ancazamfir** [comment]
> while speaking with zaki a few weeks ago he recommended stable in ci and local nightly, if you'd like nightly in ci let me know its a minor change

yes, nightly also, thanks !

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (romac) [reply .github/PULL_REQUEST_TEMPLATE.md]: I typically just have an unreleased section at the top of the changelog, but I don't mind either way. What do you think @ancazamfir?

**Comment by romac** [comment]
@marbar3778 Wow, thanks a lot! That looks great!

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (ancazamfir) [reply .github/PULL_REQUEST_TEMPLATE.md]: unreleased section at top is perfect!

**Pull request merged**